bee-crypto/src/ternary/sponge/batched_curlp/bct_curlp.rs

```diff
@@ -0,0 +1,147 @@
+// Copyright 2020 IOTA Stiftung
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except in compliance with
+// the License. You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software distributed under the License is distributed on
+// an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and limitations under the License.
+
+use crate::ternary::sponge::batched_curlp::{bct::BCTritBuf, HIGH_BITS};
+
+const NUMBER_OF_TRITS_IN_A_TRYTE: usize = 3;
```

Comment:
I know this was in the reference but this is wrong. The name suggests the input hash_length is in trytes but it's not. It's in trits. The state length just happens to be 3 times the hash length but that's all. I would remove this constant and just use 3.